tabular/tests/unittests/models/test_xt.py

```diff
@@ -24,3 +24,14 @@ def test_xt_regression(fit_helper):
     )
     dataset_name = 'ames'
     fit_helper.fit_and_validate_dataset(dataset_name=dataset_name, fit_args=fit_args)
+
+
+def test_xt_compile_binary(compile_helper):
```

Comment:
ditto, this isn't actually compiling to ONNX due to missing dependency.

Reply:
Added skl2onx into extra dependency list. Tested with a clean venv.